apps/concept-catalog/components/side-filter/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { Accordion } from '@digdir/design-system-react';
+import styles from './side-filter.module.css';
+
+const SideFilter = () => {
+  return (
+    <div className={styles.sideMenu}>
+      <Accordion className={styles.accordion}>
+        <Accordion.Item className={styles.accordionFirstItem}>
+          <Accordion.Header>Fagområde</Accordion.Header>
+          <Accordion.Content>Accordion content</Accordion.Content>
+        </Accordion.Item>
+        <Accordion.Item>
+          <Accordion.Header>Begrepsstatus</Accordion.Header>
+          <Accordion.Content>Accordion content</Accordion.Content>
+        </Accordion.Item>
+        <Accordion.Item>
+          <Accordion.Header>Tildelt</Accordion.Header>
+          <Accordion.Content>Accordion content</Accordion.Content>
+        </Accordion.Item>
+        <Accordion.Item>
+          <Accordion.Header>Publiseringstilstand</Accordion.Header>
+          <Accordion.Content>Accordion content</Accordion.Content>
+        </Accordion.Item>
+        <Accordion.Item>
+          <Accordion.Header>Navn og begrep</Accordion.Header>
+          <Accordion.Content>Accordion content</Accordion.Content>
+        </Accordion.Item>
```

Comment:
Du kan jo vurdere å flytte disse verdiene over i ei liste, og iterere over de i en løkke for Accordion.Item

Reply:
Godt Forslag. 👍